src/main/java/com/powsybl/openloadflow/sa/AcSecurityAnalysis.java

```diff
@@ -173,6 +239,22 @@ private SecurityAnalysisResult runSimulations(LfNetwork network, List<Propagated
                                                                                          preContingencyNetworkResult, createResultExtension);
                                 postContingencyResults.add(postContingencyResult);
 
+                                List<OperatorStrategy> operatorStrategiesForThisContingency = operatorStrategiesByContingencyId.get(lfContingency.getId());
+                                if (operatorStrategiesForThisContingency != null) {
+                                    // we have at least an operator strategy for this contingency.
+                                    if (operatorStrategiesForThisContingency.size() == 1) {
+                                        Optional<OperatorStrategyResult> optionalOperatorStrategyResult = runActionSimulation(network, context, propagatedContingency.getContingency(),
+                                                operatorStrategiesForThisContingency.get(0), preContingencyLimitViolationManager,
+                                                securityAnalysisParameters.getIncreasedViolationsParameters(), postContingencyResult.getLimitViolationsResult(), lfActionById,
+                                                preContingencyNetworkResult, createResultExtension, allSwitchesToClose.stream().map(Switch::getId).collect(Collectors.toList()));
+                                        if (optionalOperatorStrategyResult.isPresent()) {
+                                            operatorStrategyResults.add(optionalOperatorStrategyResult.get());
+                                        }
+                                    } else {
+                                        LOGGER.warn("A contingency has several operator strategies: not supported yet");
```

Comment:
Do you plan to implement it in this PR?

Reply:
No, because it requires a save/restore of the network state just after contingency.

Reply:
And why is it a problem? Is there something different to do that what we are doing for contingencies?